src/docs/release/breaking-changes/material-chip-button-semantics.md

```diff
@@ -1,126 +1,60 @@
 ---
-title: Material Chip Button Semantics description: Brief description similar to
-the "context" section below.
+title: Material Chip button semantics
+description: Interactive Material Chips are now semantically marked as buttons.
 ---
 
 ## Summary
 
 Flutter now applies the semantic label of `button` to all interactive [Material
-Chips](https://material.io/components/chips) for accessibility purposes.
+Chips][] for accessibility purposes.
 
 ## Context
 
-Interactive Material Chips (namely
-[ActionChip](https://api.flutter.dev/flutter/material/ActionChip-class.html),
-[ChoiceChip](https://api.flutter.dev/flutter/material/ChoiceChip-class.html),
-[FilterChip](https://api.flutter.dev/flutter/material/FilterChip-class.html),
-and [InputChip](https://api.flutter.dev/flutter/material/InputChip-class.html))
-are now semantically marked as being buttons. Please note that the
-non-interactive information
-[Chip](https://api.flutter.dev/flutter/material/Chip-class.html) is not.
-
-This will allow accessibility tools, search engines, and other semantic analysis
-software to determine the meaning of the application. For example, it will allow
-screen readers (such as TalkBack on Android and VoiceOver on iOS) to announce a
-tappable Chip as a "button", which can assist users in navigating the
-application. Prior to this change, users of accessibility tools may have had a
-subpar experience, unless application developers implemented a workaround by
-manually adding the missing semantics to the Chips in their applications.
+Interactive Material Chips (namely [`ActionChip`][], [`ChoiceChip`][],
+[`FilterChip`][], and [`InputChip`][]) are now semantically marked as being
+buttons. However, the non-interactive information [`Chip`][] is not.
+
+Marking Chips as buttons helps accessibility tools, search engines, and other
+semantic analysis software understand the meaning of an app. For example, it
+allows screen readers (such as TalkBack on Android and VoiceOver on iOS) to
+announce a tappable Chip as a "button", which can assist users in navigating
+your app. Prior to this change, users of accessibility tools may have had a
+subpar experience, unless you implemented a workaround by manually adding the
+missing semantics to the Chip widgets in your app.
 
 ## Description of change
 
-In `package:flutter/material.dart`, the `build(...)` method of the
-`_RawChipState` class (used by all chips) returns the resulting built chip
-widget wrapped in a
-[Semantics](https://api.flutter.dev/flutter/widgets/Semantics-class.html)
-widget.
+The outermost [`Semantics`][] widget which wraps all Chip classes to describe
```

Comment:
When you see "which" without a comma, as a rule either insert a comma or replace it with "that". https://developers.google.com/style/commas#setting-off-other-kinds-of-clauses 

```suggestion
The outermost [`Semantics`][] widget that wraps all Chip classes to describe
```